enclave-runtime/src/tests.rs

```diff
@@ -377,7 +381,12 @@ fn test_create_block_and_confirmation_works() {
 	ensure_no_empty_shard_directory_exists();
 
 	// create top pool
-	unsafe { rpc::worker_api_direct::initialize_pool() };
+	let chain_api = Arc::new(SideChainApi::<itp_types::Block>::new());
+	let top_pool = TopPoolContainer::new(BasicPool::create(
+		Default::default(),
+		chain_api,
+		Arc::new(TestRpcResponder::new()),
+	));
```

Comment:
In these tests we can now use the local top pool container, instead of having to juggle the global top pool and hope it all goes well when running the tests in parallel.